src/common/snippets/tests/src/pass/mha_tokenization.cpp

```diff
@@ -168,6 +168,7 @@ TEST_F(TokenizeMHASnippetsTests, smoke_Snippets_MHA3D_SplitM) {
     run();
 }
 
+#ifdef NDEBUG // Issue TBD
```

Comment:
and only for Windows?
Typically, test is marked as disabled instead compiling out

Reply:
We do not test Linux debug builds) Disabled for windows only for a while